CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 0.0.100 - WIP
+
+## Serialization Compatibility
+ * HTLCs which were in the process of being claimed on-chain when a pre-0.0.100
+   `ChannelMonitor` was serialized may generate `PaymentForwarded` events with
+   spurious `fee_earned_msat` values. This only applies to payments which were
+   unresolved at the time of the upgrade.
+ * 0.0.100 clients with pending PaymentForwarded events at serialization-time
+   will generate serialized `ChannelManager` objects which 0.0.99 and earlier
+   clients cannot read. The likelihood of this can be reduced by ensuring you
+   process all pending events immediately before serialization.
+
+
```

Comment:
Mentions e00bf8da serialization relaxation? Maybe we'll have users eager to experiment with the odd type semantic

Reply:
I don't believe you can currently (reasonably) serialize events in a way that LDK cares about - I believe we only (really) read them as a part of a `ChannelManager` or `ChannelMonitor`, so in order for users to get anything out of it they'd need to be editing `ChannelManager`s in the middle of the serialization logic.